### PR TITLE
Cleanup Portal

### LINKED ISSUE #3237: Fix or remove language
I was checking the code
https://github.com/plone/Products.CMFPlone/blob/36a66a82d8ff975148634626a70fd8b01f7e95b9/Products/CMFPlone/Portal.py#L150-L157

`cmp` should not be used because it was removed in Python3.

### PR DESCRIPTION
**Pull request by jensens**
Minor cleanups and removal of dead code. 

Closes #3237

### CODE REVIEW

File: Products/CMFPlone/Portal.py [hunk 1]
```diff
@@ -207,17 +202,6 @@ def folderlistingFolderContents(self, contentFilter=None):
         """
         return self.listFolderContents(contentFilter)
 
-    security.declarePublic('availableLanguages')
-
-    def availableLanguages(self):
```
Comment (ale-rt) [inline Products/CMFPlone/Portal.py:212]: This will fix #3237

**Comment by ale-rt** [comment]
@jenkins-plone-org please run jobs

**Review by ale-rt — CHANGES_REQUESTED** [review]
I see this is still a draft.
Maybe you plan to add more to it.
Anyway if the tests are green I would merge it.
A towncrier entry 3237.bugfix would be appreciated :)

File: Products/CMFPlone/Portal.py (hunk 1 above)
Comment (jensens) [reply Products/CMFPlone/Portal.py:212]: indeed :-)

**Comment by jensens** [comment]
yes, I create them mostly as draft and make them ready if all set. Now ready!

**Comment by jensens** [comment]
@jenkins-plone-org please run jobs

**Comment by jensens** [comment]
@jenkins-plone-org please run jobs

**Review by ale-rt — APPROVED** [review]

**Pull request merged**